index.html

```diff
@@ -0,0 +1,319 @@
+            <div class="left-block">
+                <div class="pt">
+                    <h2>Multi-Post</br>
+                        Stories</h2>
+                    <ul class="frame2-l">
+                        <li class="client">
+                            <h5 class="canopy">CANOPY</h5>
+                        </li>
+                        <li class="dot"></li>
+                        <li class="role">
+                            <h5 class="Back">Back End Dev</h5>
+                        </li>
+                        <li class="dot"></li>
+                        <li class="year-label">
+                            <h5 class="year">2015</h5>
+                        </li>
+                    </ul>
+                    <p class="pry-text">A daily selection of privately personalized reads; no accounts or sign-ups
+                        required.</p>
+                    <ul class="tags">
+                        <li class="tag1">
+                            <a href="#">
+                                <p class="html">html</p>
+                            </a>
+                        </li>
+                        <li class="tag2">
+                            <a href="#">
+                                <p class="css">css</p>
+                            </a>
+                        </li>
+                        <li class="tag3">
+                            <a href="#">
+                                <p class="js">javaScript</p>
+                            </a>
+                        </li>
+                    </ul>
+
+                </div>
+
+                <div class="action">
+                    <button class="see-project-l">
+                        <p class="see-p">See Project</p>
+                    </button>
+                </div>
+            </div>
+        </div>
+    </section>
+
+    <section class="about-myself">
+        <div class="top">
+            <h2 class="myself">About <br>Myself</h2>
+            <p class="txt">Hello I’m a software developer! I can help you build a product , feature or website Look
+                through some of my work and experience! If you like what you see and have a project you need coded,
+                don’t hestiate to contact me.</p>
+            <div class="supporting-text">
+                <p class="connect">LET’S CONNECT</p>
+            </div>
+
+            <ul class="frame1">
+                <li><a href="#"><img class="github2" src="images/github.png"></a></li>
+                <li><a href="#"><img class="linkedin2" src="images/Linkedin icon.png"></a></li>
+                <li><a href="#"><img class="vector2" src="images/Vector.png"></a></li>
+                <li><a href="#"><img class="twitter2" src="images/twitter.png"></a></li>
+                <li><a href="#"><img class="m2" src="images/m.png"></a></li>
+            </ul>
+
+            <button class="resume">
+                <p class="see-resume">Get my resume</p>
+            </button>
```

Comment:
- [ ] Please use all button effects as I mentioned above :point_up: . So kindly fix this also.